gtsam/hybrid/tests/testHybridGaussianFactorGraph.cpp

```diff
@@ -327,10 +325,9 @@ TEST(HybridGaussianFactorGraph, Switching) {
     std::transform(naturalX.begin(), naturalX.end(), std::back_inserter(ordX),
                    [](int x) { return X(x); });
 
-    KeyVector ndX;
-    std::vector<int> lvls;
-    std::tie(ndX, lvls) = makeBinaryOrdering(ordX);
+    auto [ndX, lvls] = makeBinaryOrdering(ordX);
     std::copy(ndX.begin(), ndX.end(), std::back_inserter(ordering));
+    // TODO(dellaert): this has no effect!
```

Comment:
@ProfFan I think this might be you, no idea what you were trying to achieve here as `lvls` is not visible below.